test_resource/cyclo_gate.py

```diff
@@ -0,0 +1,129 @@
+"""Koninklijke Philips N.V., 2019 - 2020. All rights reserved."""
```

Comment:
Thank you for putting this copyright header.

Should we add this to all the files if not present already?

Reply:
Accepted. Added the header in all files